KappaSigmaMu/ksm-app
# Add and config Cypress

Hello :rabbit: 

This is a PR to add and config cypress to the project, already using typescript. Also, a first test example was made.

## How to test
1. Run `yarn` to install the new dependency
2. Run `yarn cy:open`

- [ ] Cypress should open and the tests should pass

PS: Cypress uses the default `localhost:3000/ksm-app` address (see `cypress.json`). If you change the port, the tests will fail.
